### PR TITLE
tests/sce: Add test for SCE in DS

### PR DESCRIPTION
**Pull request by ybznek**
Test for SCE in DS, will FAIL in maint-1.2 branch.
Wais for https://github.com/OpenSCAP/openscap/pull/349

### CODE REVIEW

File: tests/sce/test_sce_in_ds.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+test_init "test_sce_in_ds.log"
+
+test_run "SCE in DS" test_sce_in_ds test_sce_in_ds.xml
+
+test_exit
+
+
+
+
+
```
Comment (mpreisler) [inline tests/sce/test_sce_in_ds.sh]: whitespace

File: tests/sce/test_sce_in_ds.xml [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+		</ds:component-ref>
+	</ds:checklists>
+	<ds:checks>
+		<ds:component-ref id="scap_org.open-scap_cref_stub-oval.xml" xlink:href="#scap_org.open-scap_comp_stub-oval.xml"/>
+	</ds:checks>
+	<ds:extended-components>
+		<ds:component-ref id="scap_org.open-scap_cref_services--obsolete--disable_nfs_exports.sh" xlink:href="#scap_org.open-scap_ecomp_services--obsolete--disable_nfs_exports.sh"/>
+	</ds:extended-components>
+	</ds:data-stream>
+	<ds:component id="scap_org.open-scap_comp_stub-oval.xml" timestamp="2016-02-23T14:23:29">
```
Comment (mpreisler) [inline tests/sce/test_sce_in_ds.xml]: This component isn't necessary I think. Did you add it intentionally?

**Comment by mpreisler** [comment]
Why is this "do not merge"?

**Comment by ybznek** [comment]
@mpreisler 
Main reason is that after we will later merge changes from maint-1.0 to maint-1.2 and in maint-1.2 it will not PASS.
Waits for https://github.com/OpenSCAP/openscap/pull/349

File: tests/sce/test_sce_in_ds.xml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+		</ns0:description>
+		</ns0:Profile>
+
+			<ns0:Rule id="xccdf_org.open-scap.sce-community-content_rule_services_obsolete-disable_nfs_exports" selected="true">
+			<ns0:title xml:lang="en"></ns0:title>
+			<ns0:description xml:lang="en"></ns0:description>
+			
+			<ns0:check system="http://open-scap.org/page/SCE">
+				<ns0:check-import import-name="stdout">
+			</ns0:check-import>
```
Comment (jan-cerny) [inline tests/sce/test_sce_in_ds.xml]: This should be one element with /> at the end.

**Comment by mpreisler** [comment]
This was waiting for #349. Has anything changed? I was planning on merging it after the SDS was fixed in maint-1.2.

**Comment by isimluk** [comment]
Do not merge label has been removed.

**Comment by isimluk** [comment]
iirc, it works on maint-1.2. We perhaps need different test for #349. :)

**Pull request merged**